### ISSUE TITLE
Replace re-use with reuse

### ISSUE THREAD

**Issue by Anishka0107** [openaustralia/righttoknow#683]
I am not sure about this but I guess the word 'reuse' is better than 're-use'.
The website Right To Know uses the word 're-use' at many places. Whereas, searching it up on the internet, and in dictionaries like the Oxford Dictionary, Merriam Webster and Cambridge dictionary, I found out that it is better to use the word 'reuse'. The dictionaries do not define the word 're-use'. Although 're-use' is also correct, it is not recommended probably.

**Comment by Anishka0107** [comment]
@equivalentideas 
If the corrections are needed I would like to work on it.

**Comment by Keerti-Gautam** [comment]
Hi @Anishka0107 

Can you provide some instances of this occurrence from the website? It would be great to review those before we can try to think about corrections, I feel.

**Comment by Anishka0107** [comment]
Hi @Keerti-Gautam :)
Could you please go to the link https://www.righttoknow.org.au/help/requesting#reuse.

**Comment by Keerti-Gautam** [comment]
I think it was specifically mentioned in the Statement of IP Principles like that and is hence written verbatim (and subsequently taken in the header) in the below-quoted text (not truly sure as the link is broken)
The website of Right To Know has used 'reuse' as you mentioned in the last line of the first paragraph. ".... explicitly encourages reuse." 
Do you agree?  :smile:

**Comment by Anishka0107** [comment]
Yes, the old link is broken, so we cannot check that out. But, the link with which I have replaced it uses the word 'reuse' and not 're-use'. :smiley:

**Comment by henare** [comment]
Good catch @Anishka0107, I say go for it :+1:
